packages/cyberstorm/src/components/DropDown.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/DropDown.module.css";
+
+import * as RadixDropDown from "@radix-ui/react-dropdown-menu";
+import { SelectItemProps } from "./SelectItem";
+
+type DropDownProps = {
+  colorScheme?: "default" | "defaultDark" | "primary";
+  defaultOpen?: boolean;
+  content?: ReactNode[];
+  trigger: ReactNode;
+};
+
+export const DropDown: React.FC<DropDownProps> = React.forwardRef(
+  (props, ref) => {
+    const { colorScheme, defaultOpen, content, trigger } = props;
+
+    return (
+      <div className={styles.root} {...props} ref={ref}>
```

Comment:
On my IDE TypeScript complains about `ref`. Based on quick search this should use `React.forwardRef<HTMLDivElement, DropDownProps>`, but I'm not 100% certain about that so maybe do some proper research into the subject.

Also my Monday-morning brain can't quite recall why we're dealing with the refs in the first place? Anyhow settings the ref to the outermost wrapper seems odd, since that should be accessible by parents anyway? Isn't the point of the ref to attach it to some leaf component that can't be otherwise accessed from the outside? Should Storybook contain some example where the ref is actually used in a way that provides some value?

Reply:
If you can figure a way to not use refs at all, I'll gladly use that. I couldn't, Radix wants the ref for Select's items, and the trigger etc.